src/components/Sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,71 @@
+import React ,{useReducer} from 'react'
+import { useProduct } from '../../context/filterContext';
+export default function Sidebar() {
+const { state, dispatch } = useProduct();
+const {sortBy,showInventoryAll,showFastDelivery,showCategory,showRating}=state
+return (
+<div className="ec-side-navbar">
+    <div className="ec-product-filter">
+
+        <div className="ec-first-element-side-navbar">
+            <div className="h3">Filters</div>
+            <a className="h5 italic-text" href="">Clear!</a>
+        </div>
+        <div className='h5'>Price: </div>
+        <input type="range" max={10000} step={1000} defaultValue={5000} onChange={(event)=>
+        dispatch({ type: "FILTER_BY_PRICE", payload: event.target.value })
+        }
+        />
+
+        <div className="h5">Sort By:</div>
+        <div>
+            <label className='category-label' name="sort"><input onChange={()=> {
+                dispatch({ type: "SORT", payload: "HIGH_TO_LOW" });
+                }}
+                checked={sortBy && sortBy === "HIGH_TO_LOW"}type="radio" />High-To-Low</label>
+            <label className='category-label' name="sort"><input onChange={()=> {
+                dispatch({ type: "SORT", payload: "LOW_TO_HIGH" });
+                }}
```

Comment:
Suggestion: you can fix indentation of input tag with label tag, for readability